include/libtorrent/kademlia/node_id.hpp

```diff
@@ -64,7 +64,7 @@ int TORRENT_EXTRA_EXPORT min_distance_exp(node_id const& n1, std::vector<node_id
 
 node_id TORRENT_EXTRA_EXPORT generate_id(address const& external_ip);
 node_id TORRENT_EXTRA_EXPORT generate_random_id();
-void TORRENT_EXTRA_EXPORT make_id_secret(node_id& in);
```

Comment:
I don't understand this change either. is the attribute has to come before the return type, presumably that would apply to all the other functions here too..